File: Server/NotificationModule/Models/NotificationDto.cs
using NotificationsModule.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace NotificationsModule.Models
{
    public class NotificationDto
    {
        public string? Title { get; set; }
        public string? SubTitle { get; set; }
        public NotificationType Type { get; set; }
        public long Amount { get; set; }
        public DateTime DateTime { get; set; }
    }
}

File: Client/app/src/main/java/com/nks/interactive/multimediapanel/models/notification/NotificationDto.kt
package com.nks.interactive.multimediapanel.models.notification

import java.time.LocalDateTime

data class NotificationDto(
    val title: String?,
    val subTitle: String?,
    val type: NotificationType,
    val amount: Long,
    val dateTime: LocalDateTime
)

enum class NotificationType {
    None,
    Ferry,
    Fined,
    Tollgate,
    Train
}
